frameworks/cassandra/tests/conftest.py

```diff
@@ -1,12 +1,19 @@
 import pytest
-import sdk_repository as repo
+import sdk_repository
+import sdk_security
 
 
 @pytest.fixture(scope='session')
-def configure_universe():
+def configure_session():
     stub_urls = {}
     try:
-        stub_urls = repo.add_universe_repos()
+        stub_urls = sdk_repository.add_universe_repos()
```

Comment:
Also dependencies can be managed by pytest.mark.usefixtures or autouse=true instead of explicit fixture dependencies in the tests, but I don't have a strong preference either way at this point in time